Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

error: implicit declaration of function 'tparm' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- signals.c.orig	2002-09-16 07:42:42.000000000 -0500
+++ signals.c	2020-12-07 19:14:43.000000000 -0600
@@ -25,6 +25,7 @@
 
 
 #include <signal.h>
+#include <stdlib.h>
 #include "signals.h"
 #include "display.h"
 
--- tty.c.orig	2002-09-16 07:42:45.000000000 -0500
+++ tty.c	2020-12-07 21:13:10.000000000 -0600
@@ -29,6 +29,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <errno.h>
+#include <curses.h>
 #include <termios.h>
 #include <sys/ioctl.h>
 
